### PR TITLE
releasenote: update labels to use a prefix

### PR DESCRIPTION
**Pull request by nikhita**
Fixes https://github.com/kubernetes/kubernetes/issues/24057

I did not update the `/release-note-none` command to `/release-note none` because it is deprecated.

/hold
- This will be a breaking change. If this lgty, I will document it as one but I wanted to get feedback on the PR first.
- Not sure if we can support both the old and the new labels at the same...if a PR has the release note body as `NONE`, then there is no way to know if we should add the `release-note-none` label or the `release-note/none` label. :thinking: 
- This PR updates the labels in `labels.yaml` _and_ the releasenote plugin at the same time. This would mean the labels will get updated almost immediately by the `label_sync` cronjob but the releasenote plugin won't use these new labels until the next bump. Suggestions on how to mitigate this?

/sig contributor-experience
/cc @cblecker @spiffxp

### CODE REVIEW

File: prow/plugins/releasenote/releasenote.go [hunk 1]
```diff
@@ -39,8 +39,8 @@ const (
 	// merge.
 	ReleaseNoteLabelNeeded    = "do-not-merge/release-note-label-needed"
 	releaseNote               = "release-note"
-	releaseNoteNone           = "release-note-none"
-	releaseNoteActionRequired = "release-note-action-required"
+	releaseNoteNone           = "release-note/none"
+	releaseNoteActionRequired = "release-note/action-required"
```
Comment (spiffxp) [inline prow/plugins/releasenote/releasenote.go]: So as far as supporting both labels at the same time goes, the pattern I saw when `mungegithub` had to do this in the past was keep the old label name around in vars prefixed with `deprecated`, eg:`deprecatedReleaseNoteNone` and `deprecatedReleaseNoteActionRequired`

then anything that _read_ the label checked for both old and new, and anything that _wrote_ the label only used the new label

**Comment by tpepper** [comment]
This change touches on

/area release-eng
/sig release

File: prow/plugins/releasenote/releasenote.go (hunk 1 above)
Comment (stevekuznetsov) [reply prow/plugins/releasenote/releasenote.go]: Doesn't `label_sync` have support for migrating labels? I don't think we need to do that

**Comment by nikhita** [comment]
Going to revisit if we really want to do this, and then reopen with changes.

**Pull request closed without merging**